sparkpost.md

```diff
@@ -1,31 +1,24 @@
 # SparkPost
 
-:warning:
-**Never ever set up a *\*.renuoapp.ch* as a sending domain to a new created account. This will block the account on SparkPost.
-If you need this, choose the second option.** :warning:
-
-There are two options to set up SparkPost:
+Each app is using a separate subaccount under the main SparkPost account sparkpost+renuoapp@renuo.ch.
 
-* With a domain for the project and a separate SparkPost account (preferred, if possible)
-* Without an exisiting domain ==> via renuoapp.ch SparkPost account
+There are two paths when setting up the project on SparkPost:
 
-**Note:** With the second option, you are only able to send mails with **\*@renuoapp.ch** as your mail sender.
+* The domain is known: it should be set up and verified under the subaccount's sending domains.
+* The domain is not known: the emails will be sent with **\*@renuoapp.ch** as your mail sender
```

Comment:
i guess that was exactly the problem, that we sent emails on test and develop and therefore the account was blocked. this would affect the main account right?

Reply:
true, if @renuoapp.ch is sending fishy content, this could affect others.
The only thing that comes to my mind would be to keep sparkpost+renuoapp@renuo.ch for develop / testing and have a separate, sparkpost@renuo.ch, for production